lib/Models/CsvCatalogItem.js

```diff
@@ -53,14 +55,15 @@ var CsvCatalogItem = function(terria, url) {
 
     this._clockTickUnsubscribe = undefined;
 
-    this._regionWmsMap = undefined;
-    this._regionDescriptor = undefined;
     this._regionMapped = false;
 
     this._csvDataset = undefined;
 
     this._clockTickUnsubscribe = undefined;
 
+    // a function that returns a colour for a given index.
+    this.colorFunc = undefined;
```

Comment:
I don't think this should be a public property.  Public properties get serialized for sharing, for one thing.